plone/app/event/tests/base_setup.py

```diff
@@ -45,8 +46,9 @@ def make_dates(self):
         past = self.past = tz.normalize(now - timedelta(days=10))
         future = self.future = tz.normalize(now + timedelta(days=10))
         far = self.far = tz.normalize(now + timedelta(days=30))
+        scifi = self.scifi = tz.normalize(now + relativedelta(years=50))
```

Comment:
I'd go with `now + 50 * timedelta(days=365)` and remove the dateutil import above

Reply:
but this is a little bit incorrect (leap year), this year have 366 days ;-)

Reply:
but for the test it could be okay

Reply:
If you insist, you can add `dateutil` to `setup.py` in `[test]` extra_require ... but I do not know if the dependency checker is smart enough for "test dependencies" only ...